customize-controls.js

```diff
@@ -58,7 +60,11 @@ var CustomizerBrowserHistory = (function( api, $ ) {
 		} );
 		api.section.each( function( section ) {
 			if ( section.active() && section.expanded() ) {
-				expandedSection = section.id;
+				if ( 'outer' === section.params.type ) {
```

Comment:
I think the condition in here should instead be `api.OuterSection && section.extended( api.OuterSection )`. This would then ensure that if someone subclasses `OuterSection` and gives it a different `type` that it will continue to work.